core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -453,6 +455,11 @@ private List<MediaBrowserCompat.MediaItem> loadChildrenSynchronous(@NonNull Stri
                 }
             }
         }
+
+        // We are connected to the car, Android Auto is buggy and not sending the broadcast message
+        Intent i = new Intent("com.google.android.gms.car.media.STATUS");
+        i.putExtra("media_connection_status", "media_connected");
+        sendBroadcast(i);
```

Comment:
Media browsing might happen for other reasons as well. For example, some devices seem to query the media browser after starting the device to restore the player notification. We don't want the app to randomly start playing in callbacks that are not supposed to start playing...

Reply:
I agree this is a hack, let me close this PR and look into other solutions